README.md

```diff
@@ -1 +1,211 @@
-# Math-Magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math-Magicians] <a name="about-project"></a>
+
+> The Math-Magician project is a calculator application that enables users to perform basic arithmetic calculations. Developed using React, a widely used JavaScript library for creating user interfaces, this Single Page App (SPA) also provides a random math-related quote for users to read.
+
+## 🛠 Built With <a name="React"></a>
+
+1. React
+2. HTML.
+3. CSS.
+4. JAVASCRIPT
+5. LINTERS.
+6. ES6
+7. Webpack
+8. GitFlow
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">HTML</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="http://www.css.com/">CSS</a></li>
+    <li><a href="http://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+  </ul>
```

Comment:
- [ ] Some of the tech stacks listed here were not used in this project. Kindly remove them. 👍